portfolio/src/main/java/com/google/sps/data/Comment.java

```diff
@@ -5,22 +5,27 @@
     private String username;
     private String message;
     private String date;
+    private String emoji;
```

Comment:
Because your Comment logic is nicely separated into its own class, now it's easy to add more functionality to it such as emoji and score. Nice to see the benefits of good code!

Reply:
Thanks!